src/Controller/GameController.php

```diff
@@ -37,10 +37,11 @@ public function new()
     public function show(Request $request)
     {
         $playerHash = $request->cookies->get($this::COOKIE_KEY);
-        if(!$playerHash){
+        if (!$playerHash || !$this->session->get($playerHash)) {
```

Comment:
it has been discussed, but it can be improved by using listeners. (https://symfony.com/doc/current/event_dispatcher/before_after_filters.html)